scripts/o3de/ExportScripts/export_source_ios_xcode.py

```diff
@@ -84,9 +83,9 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                     add_help=False
         )
 
-        default_tools_path = o3de_context.project_path / 'build/Mac'
-        default_ios_path = o3de_context.project_path / 'build/iOS'
-        parser.add_argument(exp.CUSTOM_SCRIPT_HELP_ARGUMENT,default=False,action='store_true',help='Show this help message and exit.')
+        default_tools_path = o3de_context.project_path / 'build/mac_xcode'
+        default_ios_path = o3de_context.project_path / 'build/ios_mono'
```

Comment:
The nomenclature here seems inconsistent. The `mac` uses a generator-specific suffix, while the `ios` uses a built type suffix. Why not use `tools` instead of `mac_xcode` for the tools path, and `game_ios` for the ios path?

Reply:
Well I guess it could be [ios_xcode_mono ](https://github.com/o3de/o3de/blob/development/cmake/Platform/iOS/CMakePresets.json#L28)

Reply:
i think `_mono` implies that you can have a non-mono, which isn't the case

Reply:
On iOS it is possible to have non-mono.
It won't work, but the permutation is there.

Reply:
We actually do support non-monolithic for iOS. However, last time i checked, I think release builds disable any module loading anyways and will only work in monolithic mode. The other reason is for consistency, since both windows and linux do not use this nomenclature.